File: python/hamming/hamming.py
def distance(strand_a, strand_b):

    if len(strand_a) == len(strand_b):
        hamming_distance = 0
        for letter_a, letter_b in zip(strand_a, strand_b):
            if not letter_a == letter_b:
                hamming_distance += 1
        return hamming_distance
    else:
        raise ValueError('.+')

File: php/hamming/hamming.php
<?php

function distance(string $strandA, string $strandB) : int {

    $lengthA = strlen($strandA);
    $lengthB = strlen($strandB);

    if  ($lengthA == $lengthB) {
        $hamming_distance = 0;
        for ($i=0; $i<$lengthA; $i++) {
            if ($strandA[$i] != $strandB[$i]) {
                $hamming_distance++;
            }
        }

        return $hamming_distance;
    }

    throw new InvalidArgumentException("DNA strands must be of equal length.");
}
